### ISSUE TITLE
Добавить ADC хаблист

### ISSUE THREAD

**Issue by Tantrix** [pavel-pimenov/flylinkdc-r5xx#1735]
https://dchublist.org/adchublist.xml.bz2

**Comment by HadesDCH** [comment]
Yeah that would make sense - flilink suports adc/adcs yet has no hublists that do list adc hubs.
Hublist can be reached without ssl too if flylink doesnt support that.
http://dchublist.org/hublist.xml.bz2   <== this is a full list - why limit it to adc only.
